python/server/qgsserverprojectutils.sip

```diff
@@ -243,5 +243,5 @@ namespace QgsServerProjectUtils
       * @param project the QGIS project
       * @return the Layer ids list.
       */
-    QStringList wcsLayers( const QgsProject &project );
+    QStringList wcsLayerIds( const QgsProject &project );
```

Comment:
In the WMS context, the ids is used to know if the layer is part of published WCS layers. I think that a second method to return a QList< QgsRasterLayer *> could be useful.